### PR TITLE
0.4.0

### PR DESCRIPTION
**Pull request by Morganamilo**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ terraform {
 
   required_providers {
     ably = {
-      source = "ably/ably"
+      source = "hashicorp/ably"
```
Comment (Morganamilo) [inline docs/index.md:40]: Huh any one know what happened here? I think running tfplugindocs made this change. Did the provider move to under hashicorp?

Comment (graham-russell) [reply docs/index.md:40]: I think this one should remain ably/ably.

Comment (Morganamilo) [reply docs/index.md:40]: Our examples use `hashicorp/ably` too so this may be intentional? @kavalerov

Comment (graham-russell) [reply docs/index.md:40]: It should definitely be ably/ably as we don't host our provider in the hashicorp namespace on the terraform registry. 
We should update the examples as well in this case as they're incorrect. 
Probably a relic from when we created the examples before working on provider implementation.

Comment (Morganamilo) [reply docs/index.md:40]: 534d7565 Shows what may be a manual update to the file. Need to see how to set it to ably and stick

Comment (Morganamilo) [reply docs/index.md]: So it seems to read the examples. Changing the examples fixed the issue.

**Review by graham-russell — APPROVED** [review]

**Pull request merged**